.github/workflows/pr-libbpfgo.yaml

```diff
@@ -5,15 +5,18 @@ on:
       - "libbpfgo/**"
 jobs:
   build-test:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
```

Comment:
Is this a secret stored somewhere? I'm just curious where the value comes from.

Reply:
Ah nvm https://github.com/aquasecurity/tracee/pull/693/files#diff-b5f15b3b6c9c91e9bede2818583e85ac47b2f7efc6331846a73daf3b566c3578R11 must be this right? 

For my understanding, how does it know where the self hosted machine is located?

Reply:
Yes that is it. That label (centos-8.2-linux-4.18) corresponds to the machine which was setup in the settings of the repo. ([docs here](https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners) if you're curious)